Revamp docs for Docathon (#225)
pulumi/pulumi-eks: README.md

Pulumi's library for easily creating and managing EKS Kubernetes clusters.

> This package is meant for use with the Pulumi CLI. Please visit [pulumi.io](https://pulumi.io) for

Reviewer: ```suggestion
This package is meant for use with the Pulumi CLI. Please visit [pulumi.com](https://www.pulumi.com) for
```

1. [Configure AWS Credentials](https://www.pulumi.com/docs/reference/clouds/aws/setup/).
1. [Install AWS IAM Authenticator for Kubernetes](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html).
1. [Install `kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl).

## Installing

This package is available in JavaScript/TypeScript for use with Node.js.  Install it using either `npm`:

```bash
$ npm install @pulumi/eks
```

or `yarn`:

```bash
$ yarn add @pulumi/eks
```

This package requires `aws-iam-authenticator` for Amazon EKS in order to deploy resources to EKS clusters. Install this
tool using the [official instructions](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html).

## Concepts

This package provides a Pulumi component that creates and manages the resource necessary to run an EKS Kubernetes
cluster in AWS. This includes:
- The EKS cluster
- The cluster's worker nodes, and node groups which are managed by an autoscaling group
- The Kubernetes resources needed to allow the worker nodes to access the cluster

> The default configuration targets the AWS account's default VPC, and creates an autoscaling group of 2 `t2.medium` EC2 instances:

Reviewer: ```suggestion
The default configuration targets the AWS account's default VPC, and creates an autoscaling group of two `t2.medium` EC2 instances:
```

```typescript
import * as eks from "@pulumi/eks";

// Create an EKS cluster with the default configuration.
const cluster = new eks.Cluster("my-cluster");

// Export the cluster's kubeconfig.
export const kubeconfig = cluster.kubeconfig;
```

> Once the cluster is created, you can deploy into the cluster using the [`@pulumi/kubernetes`][pulumi-kubernetes] SDK, kubectl, Helm, etc.

Reviewer: ```suggestion
Once the cluster is created, you can deploy to the cluster using the [`@pulumi/kubernetes`][pulumi-kubernetes] SDK, kubectl, Helm, etc.
```
